docs/users-guide/04-asking-questions.md

```diff
@@ -107,30 +143,68 @@ Some grouping columns will give you the option of choosing how big or small to m
 
 ![Multiple groupings](./images/notebook/histogram-bins.png)
 
-Once you're done setting your metrics and groupings, click Done to close the Summarize sidebar and see your results in all their glory.
+Once you're done setting your metrics and groupings, click **Done** to close the Summarize sidebar and see your results in all their glory.
+
+## Visualizing your data
 
 If you want to jump ahead and learn about [how to change the visualization](05-visualizing-results.md) of your results, by all means, feel free.
 
+## Returning to the notebook editor
+
+To return to the notebook editor for a question, click on the show editor button in the upper right.
+
+![Show editor](./images/asking-questions/show-editor.png)
+
 ## Viewing an individual record's details
 
-Click on a record's ID number (or primary key) to see more information about a given user, order, venue, etc. You can see all fields related to that one record and all connected tables that are hidden in the table view for the sake of readability. Press the right or left arrow keys, or click on the arrows to the right or left of the screen to page through the other records in the current list.
+To see more info about a given record (a user, order, venue, etc), click on a record's ID number (or primary key). You can see all fields related to that one record and all connected tables that are hidden in the table view for the sake of readability. To page through the other records in the current table, press the right or left arrow keys, or click on the arrows to the right or left of the screen.
 
 ![Record details](./images/notebook/record-details.png)
 
-## Starting new explorations from saved questions
+## Custom expressions
+
+![Custom expression](./images/expressions/aggregation-expression.png)
+
+Custom expressions allow you to use spreadsheet-like functions and simple arithmetic within or between aggregation functions. For example, you could do `Average(sqrt[FieldX]) + Sum([FieldY])` or `Max(floor([FieldX] - [FieldY]))`, where `FieldX` and `FieldY` are fields in the currently selected table. [Learn more about writing expressions](./expressions.md)
+
+### Creating custom columns
+
+![Custom column](./images/expressions/custom-column.png)
+
+Custom columns are helpful when you need to create a new column based on a calculation, such as subtracting the value of one column from another, or extracting a portion of an existing text column. Note that columns you add in a custom question are not permanently added to your table; they'll only be present in the given question.
+
+You can use the following math operators in your formulas: `+`, `–`, `*` (multiplication), and `/` (division), along with a whole host of spreadsheet-like functions. You can also use parentheses to clarify the order of operations. You can [learn more about writing expressions here](./expressions.md).
```

Comment:
```suggestion
You can use the following math operators in your formulas: `+`, `–`, `*` (multiplication), and `/` (division), along with a whole host of spreadsheet-like functions. You can also use parentheses to clarify the order of operations. 
```

Reply:
Seems redundant since this was linked right above; maybe it's useful since they're different sections but as is looks repetitive